### PR TITLE
Update Privacy section to clarify network change

### LINKED ISSUE #28: `pending`: if a beacon is already discarded by browser, should it become non-pending
One of the privacy requirement is `Beacons are only sent over the same network that was active when the beacon was registered (e.g. if the user goes offline and moves to a new network, discard pending beacons).`

So there will be some cases that users are still holding beacon object, but it's already discarded by browser.

### PR DESCRIPTION
**Pull request by mingyc**
Update Privacy section to clarify why beacon needs to be stopped when network changes (#27 #28)

Also added links to related privacy discussions: #3, #27, #30, #34
Updated beacon TTL to 30min per suggestion in https://github.com/WICG/unload-beacon/pull/16#discussion_r932857618

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Authors: [Darren Willis](https://github.com/darrenw), [Fergal Daly](https://github.com/fergald), [Ming-Ying Chung](https://github.com/mingyc) - Google
 
-[![Super-Linter](https://github.com/WICG/unload-beacon/workflows/linter.yml/badge.svg)](https://github.com/WICG/unload-beacon/actions/workflows/super-linter.yml)
+[![Super-Linter](https://github.com/WICG/unload-beacon/workflows/Lint%20Code%20Base/badge.svg)](https://github.com/WICG/unload-beacon/actions/workflows/super-linter.yml)
```
Comment (fergald) [inline README.md:5]: seems like it shouldn't be in this change.

File: README.md [hunk 2]
```diff
@@ -305,8 +305,8 @@ special support may be needed to allow extension authors to block the sending (o
 Specifically, beacons will have the following privacy requirements:
 
 * Beacons must be sent over HTTPS.
-* Beacons are only sent over the same network that was active when the beacon was registered
-  (e.g. if the user goes offline and moves to a new network, discard pending beacons).
+* Beacons must not leak information that a page should not know when it is in bfcache,
+  e.g. if network changes after a page goes into bfcache, the beacon should not be sent; if the page then goes out of bfcache, the beacon can be sent.
```
Comment (fergald) [inline README.md:309]: I could suggest

"if the page is then restored from bfcache"

**Review by fergald — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mingyc) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (mingyc) [reply README.md:5]: Sorry this is already commited to main branch. Not sure how to remove it from this PR but should be no-op on merging.

**Pull request merged**